install_config/master_node_configuration.adoc

```diff
@@ -389,6 +389,14 @@ assetConfig:
     requestTimeoutSeconds: 0
 ----
 
+|`*corsAllowedOrigins*`
+|To access the API server from a web application using a different host name, you
+must whitelist that host name by specifying `corsAllowedOrigins` in the
+configuration field or by specifying the `--cors-allowed-origins` option on
+`openshift start`. No pinning or escaping is done to the value. See
```

Comment:
I don't know what this means:
No pinning or escaping is done to the value.

Otherwise, LGTM!

Reply:
I believe it's a regular expression, rather than a simple string comparison. So `example.com` would match domains with the text `anotherexample.com` or even `examplescom`, etc. We probably want to call this out explicitly.

cc @liggitt

Reply:
Although I see there's an example if you follow the link.

Reply:
yes, the example added in https://github.com/openshift/openshift-docs/pull/8630/files explains the syntax/meaning